cli/data/playbooks/deploy/metadata.obsah.yaml

```diff
@@ -0,0 +1,54 @@
+---
+help: |
+  Install
+
+variables:
+  certificate_source:
+    help: Where certificates are coming from. Currently default Ansible role or the foreman-installer.
+    choices:
+      - default
+      - installer
+  database_mode:
+    help: Defaults to internal. Set to 'external' if using an external database.
+    choices:
+      - internal
+      - external
+  database_host:
+    help: Host of the PostgreSQL database.
+  database_port:
+    help: Port of the PostgreSQL database.
+  database_ssl_mode:
+    help: If using an SSL connection to the database, define how it should be verified.
+    choices:
+      - disable
+      - allow
+      - prefer
+      - require
+      - verify-ca
+      - verify-full
+  database_ssl_ca:
+    help: Path to the database SSL CA certificate.
+    type: AbsolutePath
+  foreman_database_name:
+    help: Name of the Foreman database.
+  foreman_database_user:
+    help: User of the Foreman database.
+  foreman_database_password:
+    help: Password for the Foreman database.
+  candlepin_database_name:
+    help: Name of the Candlepin database.
+  candlepin_database_user:
+    help: User of the Candlepin database.
+  candlepin_database_password:
+    help: Password for the Candlepin database.
+  pulp_database_name:
+    help: Name of the Pulp database.
+  pulp_database_user:
+    help: User of the Pulp database.
+  pulp_database_password:
+    help: Password for the Pulp database.
+constraints:
+  required_together:
+    - [database_ssl_mode, database_ssl_ca]
+  required_if:
+    - ['database_mode', 'external', ['database_host', 'database_port']]
```

Comment:
Why is the port required? Shouldn't it be 5432 in 99.9% of the cases (aka: it can have a sensible default)?